### PR TITLE
Update models

### PR DESCRIPTION
**Pull request by KostinP**

### CODE REVIEW

File: src/TimeManagementService.Mappers/TimeManagementService.Mappers.csproj [hunk 1]
```diff
@@ -11,4 +11,9 @@
     <ProjectReference Include="..\TimeManagementService.Models.Dto\TimeManagementService.Models.Dto.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="RequestMappers\" />
+    <Folder Include="ResponseMappers\" />
+  </ItemGroup>
+
```
Comment (djino97) [inline src/TimeManagementService.Mappers/TimeManagementService.Mappers.csproj:18]: remove

File: src/TimeManagementService.Models.Dto/TimeManagementService.Models.Dto.csproj [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
 
   <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Folder Include="Responses\" />
+  </ItemGroup>
```
Comment (djino97) [inline src/TimeManagementService.Models.Dto/TimeManagementService.Models.Dto.csproj:11]: remove

File: src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs [hunk 3]
```diff
@@ -1,71 +1,161 @@
...
+                    //{
+                    //    RuleFor(x => x.Patch.Operations)
+                    //    .UniqueOperationWithAllowedOp(WorkerUserIdPath, "add", "replace");
+
+                    //    //check exist?
+
+                    //    RuleFor(wt => (Guid)GetOperationByPath(wt.Patch, WorkerUserIdPath).value)
+                    //    .NotEmpty()
+                    //    .WithMessage($"{WorkerUserIdPath} must not be empty.");
+                    //});
```
Comment (djino97) [inline src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs:93]: comments?

File: src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs [hunk 4]
```diff
@@ -1,71 +1,161 @@
...
     public class EditWorkTimeRequestValidator : AbstractValidator<EditWorkTimeRequest>
     {
+        private static List<string> Paths
+            => new List<string> { DescriptionPath, TitlePath, ProjectIdPath,// WorkerUserIdPath,
+                StartTimePath, EndTimePath };
+
+        private static string DescriptionPath => $"/{nameof(DbWorkTime.Description)}";
+        private static string TitlePath => $"/{nameof(DbWorkTime.Title)}";
+        private static string ProjectIdPath => $"/{nameof(DbWorkTime.ProjectId)}";
+        //private static string WorkerUserIdPath => $"/{nameof(DbWorkTime.WorkerUserId)}";
```
Comment (djino97) [inline src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs:25]: remove

File: src/TimeManagementService.Mappers/TimeManagementService.Mappers.csproj (hunk 1 above)
Comment (KostinP) [reply src/TimeManagementService.Mappers/TimeManagementService.Mappers.csproj:18]: Disagree.

File: src/TimeManagementService.Models.Dto/TimeManagementService.Models.Dto.csproj (hunk 2 above)
Comment (KostinP) [reply src/TimeManagementService.Models.Dto/TimeManagementService.Models.Dto.csproj:11]: Why?

File: src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs (hunk 3 above)
Comment (KostinP) [reply src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs:93]: I am waiting for Spartak's answer to the question at the beginning of the comment

File: src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs (hunk 4 above)
Comment (KostinP) [reply src/TimeManagementService.Validation/EditWorkTimeRequestValidator.cs:25]: Linked to commented out areas

File: src/TimeManagementService.Models.Dto/Requests/WorkTimeRequest.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+{
+    public class WorkTimeRequest
+    {
+        public Guid? UserId { get; set; }
+        public DateTime StartDate { get; set; }
+        public DateTime EndDate { get; set; }
+        public int Minutes { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public Guid ProjectId { get; set; }
```
Comment (djino97) [inline src/TimeManagementService.Models.Dto/Requests/WorkTimeRequest.cs:14]: Need add new field `name task`

File: src/TimeManagementService.Models.Dto/TimeManagementService.Models.Dto.csproj [hunk 6]
```diff
@@ -1,9 +1,17 @@
...
     <TargetFramework>netstandard2.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\TimeManagementService.Models.Db\TimeManagementService.Models.Db.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Models\" />
+  </ItemGroup>
```
Comment (djino97) [inline src/TimeManagementService.Models.Dto/TimeManagementService.Models.Dto.csproj:15]: remove

**Pull request closed without merging**